xivo_dao/alchemy/user_external_app.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy.schema import Column, ForeignKey
+from sqlalchemy.types import JSON, String, Text
+
+from xivo_dao.helpers.db_manager import Base
+
+
+class UserExternalApp(Base):
+
+    __tablename__ = 'user_external_app'
+
+    name = Column(Text, primary_key=True)
+    user_uuid = Column(
+        String(38),
```

Comment:
```suggestion
        String(36),
```

Reply:
We cannot, length must be the same as the referenced key .... so sad